charts/node-analyzer/templates/NOTES.txt

```diff
@@ -0,0 +1,9 @@
+The node analyzer for Sysdig Secure DevOps Platform is spinning up on each node in your
```

Comment:
Please recommend if this Notes make sense for node analyzer chart or should there be some different information in here?

Reply:
Maybe something along [these lines](https://docs.sysdig.com/en/docs/installation/node-analyzer-multi-feature-installation/#confirm-image-analyzerhttpsdocssysdigcomendocssysdig-securescanningscan-running-imagesscan-running-images)?